### ISSUE TITLE
`Node.suffixIds` to update references also

### ISSUE THREAD

**Issue by erikvanoosten** [sparsetech/pine#52]
It would be awesome if `Node.suffixIds()` could also update id references.

For example:

```scala
val template = html"""
<label for="tags">Tags</label>
<input id="tags" type="text" aria-describedby="tags-help"/>
<p id="tags-help"></p>
"""

assert(template.suffixIds("1").toHtml === """
<label for="tags1">Tags</label>
<input id="tags1" type="text" aria-describedby="tags-help1"/>
<p id="tags-help1"></p>
""")
```

Note that the `for` and `aria-describedby` attributes are also updated. Maybe there are more 'standard' attributes that should be updated like this.

**Comment by erikvanoosten** [comment]
One idea to facilitate custom references is to take a parameter that has a list of attribute names that should be inspected.

**Comment by tindzk** [comment]
That's a good idea. We could add a second parameter which defaults to `List("id", "for")`.

**Comment by erikvanoosten** [comment]
Aha, you mean just append the suffix to the given attributes! Yes, that is much simpler then what I proposed. I like that. However, I would recommend to make the default `Seq("id")` so that it stays backward compatible (well, at least source compatible).
BTW, I found another attribute that I need to update: `list` (for dropdowns on inputs of type text).
